config/backend-config/multi-workspace-config.go

```diff
@@ -292,10 +292,11 @@ func (multiWorkspaceConfig *MultiWorkspaceConfig) makeHTTPRequest(url string) ([
 		return []byte{}, 400, err
 	}
 
-	var respBody []byte
-	if resp != nil && resp.Body != nil {
-		respBody, _ = IoUtil.ReadAll(resp.Body)
-		defer resp.Body.Close()
+	defer resp.Body.Close()
+
+	respBody, err := IoUtil.ReadAll(resp.Body)
```

Comment:
question: why we are using `IoUtil.ReadAll` instead of `io.ReadAll`

```suggestion
	respBody, err := io.ReadAll(resp.Body)
```

Reply:
done